supertokens/supertokens.go

```diff
@@ -6,17 +6,19 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
-	"reflect"
+	"strconv"
 	"strings"
 )
 
 type SuperTokens struct {
-	instance      *SuperTokens
+	Instance      *SuperTokens
```

Comment:
I don't think we should use instance inside the struct like this.. since we already have the variable `s`